test/jdk/javax/swing/ButtonGroup/TestButtonGroupFocusTraversal.java

```diff
@@ -76,6 +79,20 @@ public void run() {
                 radioButton1 = new JRadioButton("1");
                 radioButton2 = new JRadioButton("2");
```

Comment:
Just to be safe please check the JCheckBox, it also extends the JToggleButton and we should not break it.